dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListenerManager.cpp

```diff
@@ -44,6 +45,12 @@ void NotificationListenerManager::RemoveListener(std::wstring appId)
     m_notificationListeners.erase(appId);
 }
 
+bool NotificationListenerManager::HasSinkForAppId(std::wstring const& appId)
+{
+    auto it = m_notificationListeners.find(appId);
+    return it != m_notificationListeners.end();
```

Comment:
what about using m_notificationListeners.contains()?

Reply:
The .contains() API is with c++20 and we are not allowed to upgrade :(